src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -2,27 +2,27 @@ import { join, sep } from 'path';
 
 import get from 'platform/utilities/data/get';
 
-const APP_SELECTOR = '#react-root';
-const ARRAY_ITEM_SELECTOR =
+export const APP_SELECTOR = '#react-root';
+export const ARRAY_ITEM_SELECTOR =
   'div[name^="topOfTable_"] ~ div.va-growable-background';
-const FIELD_SELECTOR = 'input, select, textarea';
-const LOADING_SELECTOR = '.loading-indicator';
+export const FIELD_SELECTOR = 'input, select, textarea';
+export const LOADING_SELECTOR = '.loading-indicator';
 
 // Force interactions on elements, skipping the default checks for the
 // "user interactive" state of an element, potentially saving some time.
 // More importantly, this ensures the interaction will target the actual
 // selected element, which overrides the default behavior that simulates
 // how a real user might try to interact with a target element that has moved.
 // https://github.com/cypress-io/cypress/issues/6165
-const FORCE_OPTION = { force: true };
+export const FORCE_OPTION = { force: true };
```

Comment:
Only need this one, but went ahead and exported all of the them in case needed in future by CT e2e tests